src/libs/ajax/ajax-common.ts

```diff
@@ -228,6 +228,11 @@ export const fetchEcm = _.flow(
   withRetryAfterReloadingExpiredAuthToken
 )(fetchOk);
 
+export const fetchCbas = _.flow(
+  withUrlPrefix(`${getConfig().cbasUrlRoot}/api/batch/v1/`),
+  withRetryAfterReloadingExpiredAuthToken
+)(fetchOk);
+
```

Comment:
Is this used anywhere? In #3986 we decided to replace this with a generic fetchFromProxy so as to not duplicate across Cromwell and CBAS api calls routed through a proxy.